hivemind/utils/grpc.py

```diff
@@ -1,17 +1,151 @@
 """
 Utilities for running GRPC services: compile protobuf, patch legacy versions, etc
 """
+from __future__ import annotations
+import os
+import threading
+from typing import NamedTuple, Sequence, Tuple, Optional, Union, Any, Dict, TypeVar, Type
```

Comment:
If we're using `from __future__ import annotations`, do we need to import tuple and dict from typing?

Reply:
i agree, but i would request to keep it as is AND write a todo for #98 , ok?